docs/Process.md

```diff
@@ -0,0 +1,25 @@
+# Work Process in RepoSense
```

Comment:
Shall we simplify the whole doc as follows?

---

# Process

We generally follow the  [The OSS-Generic Reference Process](https://oss-generic.github.io/process/) but with the following deviations.

* We will squash the commits when merging a PR. Therefore, there is no need to follow a strict commit organization or write elaborate commit messages for each commit. There is no need to use CanIHasReview tool either. However, when pushing new commits to your PR branch, do tidy up _new_ commits (i.e., commits not yet pushed) e.g., squash noise commits.

* When syncing local branch with upstream, use _merge_ rather than _rebase_ to avoid the need to force push your branch.

Reply:
> use merge rather than rebase to avoid the need to force push your branch.

Hmm for merge, isn't it more difficult than rebase to revert a single commit from a particular PR?

Reply:
That's true. However, it's hard for GitHub to keep track of past comments  or "what's new?" if you keep rebasing. GitHub is optimized for merging rather than rebasing. As we are not using CIHR (which is optimized for rebasing), might as well go with what GitHub wants us to do. If it doesn't work out, we'll switch to rebase later.